mne/connectivity/cfc.py

```diff
@@ -0,0 +1,389 @@
+""" Cross frequency coupling functions. """
+# Authors: Praveen Sripad <praveen.sripad@rwth-aachen.de>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Christopher Holdgraf <choldgraf@berkeley.edu>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from numpy.random import shuffle
+from scipy import stats
+from scipy.signal import hilbert
```

Comment:
These two lines, for example, probably need to be nested within whatever function(s) use them. At the very least the `signal` import will need to be nested.